gateway-js/src/index.ts

```diff
@@ -292,47 +304,47 @@ export class ApolloGateway implements GraphQLService {
       // only using JSON.stringify on the DocumentNode (and thus doesn't account
       // for unicode characters, etc.), but it should do a reasonable job at
       // providing a caching document store for most operations.
-      maxSize: Math.pow(2, 20) * (approximateQueryPlanStoreMiB || 30),
+      maxSize: 2 ** 20 * (approximateQueryPlanStoreMiB || 30),
```

Comment:
Disagree with this rule, I'd argue it's now less readable since I have to consider order of operations